spec/binarySearchTree.js

```diff
@@ -0,0 +1,76 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import BinarySearchTree from '../src/binarySearchTree'
+
+chai.use(chaiChange)
+
+describe('BinarySearchTree', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect( BinarySearchTree ).to.be.a( 'function' )
+  })
+
+  context( 'recursiveInsert()', () => {
+    const treeOfEternalSadness = new BinarySearchTree()
```

Comment:
What is your expected behavior for inserting a value that is already present in the BST?